core/controllers/acl_decorators_test.py

```diff
@@ -3662,15 +3662,13 @@ def setUp(self):
             },
             'change to state 1')
 
-        translation_suggestion = suggestion_services.query_suggestions(
```

Comment:
We should remove this function in case it's not reliable?

Reply:
Is it that the function isn't reliable or that it gives data in an order that is not predetermined?

Reply:
Yes. I initially thought of creating relevent suggestion in the test itself, rather than in setup as it guarantees that only the required suggestion for the test is created.